comment/tests/test_models.py

```diff
@@ -211,36 +245,166 @@ def setUp(self):
     def test_reaction_count(self):
         self.assertEqual(self.comment_1.reaction.likes, 0)
         self.assertEqual(self.comment_1.reaction.dislikes, 0)
-        self.comment_1.reaction.decrease_reaction_count(ReactionInstance.ReactionType.LIKE.value)
-        self.comment_1.reaction.refresh_from_db()
-        self.assertEqual(self.comment_1.reaction.likes, 0)
-        self.comment_1.reaction.decrease_reaction_count(ReactionInstance.ReactionType.DISLIKE.value)
-        self.comment_1.reaction.refresh_from_db()
-        self.assertEqual(self.comment_1.reaction.dislikes, 0)
 
         self.comment_1.reaction.increase_reaction_count(ReactionInstance.ReactionType.LIKE.value)
         self.comment_1.reaction.refresh_from_db()
+
         self.assertEqual(self.comment_1.reaction.likes, 1)
```

Comment:
May we can use `self.LIKE` or `self.DISLIKE` rather than using 1 and 2 as reaction type.

Reply:
You are right.